cylc/flow/cfgspec/globalcfg.py

```diff
@@ -823,12 +829,17 @@
 
                 .. versionchanged:: 8.0.0
 
-                   {}``batch system command template``.
+                   {}``suite.rc[runtime][<namespace>][job]
+                   batch system command template``.
 
                 The template's ``%(job)s`` will be
                 substituted by the job file path.
             '''.format(REPLACES))
-            Conf('shell', VDR.V_STRING, '/bin/bash')
+            Conf('shell', VDR.V_STRING, '/bin/bash', desc='''
+
+                {movedfrom}
+
+            '''.format(movedfrom=MOVEDFROMJOB))
```

Comment:
By the way, is there a reason you're not using f-strings instead? Would be cleaner IMO

Reply:
No, I've changed this now.